README.md

```diff
@@ -26,7 +26,7 @@ If you don't know the difference, pick the Installer.
 
 ### Mac
 
-If you don't know the difference, pick the Intel build.
+If you don't know the difference, check About This Mac from the Apple menu. If the first item is labeled "Chip", choose the Apple Silicon build. If it is labeled "Processor", choose the Intel build. 
```

Comment:
why not just make them run a terminal command like `uname -m` or `arch`? way simpler and less confusing

Reply:
It could work but might be intimidating for beginners.

Reply:
it really shouldn't be. just need to open terminal and copy paste command

Reply:
I’m not sure if using the terminal is simpler. You can get to the about this Mac menu in one click. No need to copy and paste. 

Either way, the user has to decipher the info and choose the correct build for themselves.

Reply:
the overlap for users who don't know whether they have an apple silicon mac and users who will be intimidated by a terminal is probably high haha